Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,36 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            // Homework Lesson 1 & 2 - SilentKingdom
+            // BMI Calculator for 2 people
+
+            // First person = variable1
+            string name1 = "Tom Jefferson";
+            int age1 = 19;
+            float weight1 = 50f;
+            float height1 = 156.5f;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add the 'const' modifier to 'height1'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4832834)